addon/-private/utils/element/get-scaled-client-rect.js

```diff
@@ -0,0 +1,10 @@
+export default function getScaledClientRect(element, scale) {
+  const rect = element.getBoundingClientRect();
```

Comment:
we should return rect directly if scale === `1` which is the most common case.

Reply:
good point, will update